### ISSUE TITLE
Second Child Window Placement possible?

### ISSUE THREAD

**Issue by RobinRuf** [nikitabobko/AeroSpace#596]
Hey guys,

Is there a way (or is it planned) to enable `window_placement = second child` like in yabai or many tiling window managers for linux?

To explain for those that don't know what I mean: You start a new window. When you open another one, it splits horizontally, of course. If you open a third one, it will split this half vertically, where you had the focus while opening the third window - and so on.

**Comment by hwamil** [comment]
I think once you have three you have to make the choice to join windows together in a container and each container would occupy their respective estate and would have the effects that you are seeking but there is no automatic configuration for this.

**Comment by danfbfern** [comment]
@hwamil the fact that you "have" to make the choice is probably the reason that @RobinRuf created the ticket, which is totally understandable.
Coming from Yabai or other tiling managers, this automatic management of the children is something that aerospace should also possibly do.

So currently if you have two windows side by side like this:
```
h_tiles
├── window 1
└── window 2
```
When opening a third window, the current behaviour is opening it in the same container, 3 windows side by side like this:
```
h_tiles
├── window 1
├── window 2
└── window 3
```
And this is acceptable and might actually be the preferred option if its being used in a ultrawide monitor.
But for the regular monitors like the built-in display, splitting the second and third window vertically would make more sense (OR having to accordion those two) but ideally in that monitor we could have something like this:
```
h_tiles
├── window 1
└── v_tiles
    ├── window 2
    └── window 3
```

This is possible by joining the window 2 and window 3 after opening the window 3, but there could be an option to automatically join that new window.

**Comment by hwamil** [comment]
@danfbfern Yeah, I was just answering his question. I'm a layman so that's all I can do :(
